src/view/BeeUI.py

```diff
@@ -57,7 +58,7 @@ def __init__(self):
         # Define the window itself
         self.root = tk.Tk()
         # Determine what the window will look like and what it does on close.
-        self.root.geometry("900x600")
+        self.root.geometry("1100x700")
```

Comment:
is this necessary?  It will break a few things.

Reply:
ok, I have an issue with this. For some reason, pyautogui breaks our set size of the frame, idk why

Reply:
It appears that PyAutoGUI forces emojis and labels to be larger?  Might be a side effect we will have to deal with.  Also I may be mistaken in the breaking things, it doesn't appear to cause too many issues.

Reply:
It made them smaller but the issue was the frame was small and the welcome was cutoff a little bit. I did some digging and pyautogui does a api call to some dpiAware function when imported and any calls to that api after the import are blocked. Ill revert to old size and we can change things around later

Reply:
leave 1100x700 it's fine that way.  appears to be ok everywhere